### ISSUE TITLE
Which of your training data would you recommend for segmenting cell cytoplasm from intercellular substance/glass?

### ISSUE THREAD

**Issue by michalstepniewski** [AllenCellModeling/pytorch_fnet#170]
Hello,
I did the tutorial and after 50k steps fnet is able to 'in-silico' label the nuclei. However my use case is that I want to be able to segment cells (cytoplasm and nucleus) from the surrounding intercellular substance or whatever is in-between those cells. I downloaded beta-actin and membrane caax training data however it seems to me that in the images i see the cells are tightly packed leaving little if any space between them. This is not my use case and I would first want to test Your method on one of your datasets. Can you recommend any particular data?

Sincerely,
Michal Stepniewski

**Comment by fcollman** [comment]
Do you have the ability to acquire training data yourself? meaning dye cells and take dual channel stacks so you can later only acquire single channel transmitted light stacks?

**Comment by michalstepniewski** [comment]
Well, myself not. I would have to reach out to others hoping they could lend me some data. My concern is if I use the data You used for training fnet used for the paper I should be able to reproduce Your results after given amount of steps. Otherwise there is more than one sources of error:
 1) acquisition and processing process
2) my application of the fnet
3) fnet is not quranteed to perform on this particular task after given amount of steps...
etc.
For now I am looking through Your training data :)

**Comment by fcollman** [comment]
I would caution that images that are taken on a different microscope of different cells with potentially different z spacing,  different xy resolution, different interslice timing are not likely to give good results with models trained on the data provided with the paper.  You are of course welcome to try, but there are very good reasons to imagine it will not work, and certainly will not work as well as if you acquired your own training data for your own microscope situation.

**Comment by fcollman** [comment]
so I think i better understand your question now.. you would like a pointer to a dataset with sparser cells but nucleus and membrane labels so you can try to do segmentation on those data, so to inform whether this idea will work in another context.

**Comment by michalstepniewski** [comment]
Yes. Pointer to a dataset would be awesome. I am searching for it myself but I guess I do not have enough know-how to find one fast :). Thank You for Your feedback regarding the need to train the model on the same microscope situation that will occur in the images I would like to predict :) This is very important. However will the training time decrease if I use the previously trained model (on different data) comparing to training model from scratch? I mean: does transfer learning apply here :)

**Comment by michalstepniewski** [comment]
Let's say that having done Your tutorial I now want to train the algorithm to recognize cellular membrane. I take the file, e.g.:
CAAX_100X_20171024_1-Scene-01-P1-B02.czi

Do i need to preprocess it somehow before I can use it to train fnet? I see that the files used in example have 70 slices and seven channels stacked slice1(c1, c2 .. c7), slice2(c1, c2 .. c7). Do I need to add the dummy channel and cut off some z slices? (run some sort of interpolation across z)? Is there any settings I can/should change to be able to use the czi file as is? Do I need to rescale channel intensities (I noticed some channels from the example data are brighter than in the CAAX_100X_20171024_1-Scene-01-P1-B02.czi).
Thank You in advance,
Sincerely,
Michal Stepniewski
